docs/data/data-core-blocks.md

```diff
@@ -72,6 +72,38 @@ Returns an array with the child blocks of a given block.
 
 Array of child block names.
 
+### getBlockSupport
+
+Returns the block support value for a feature, if defined.
+
+*Parameters*
+
+ * state: Data state.
+ * nameOrType: Block name or type object
+ * feature: Feature to retrieve
+ * defaultSupports: Default value to return if not
+                                          explicitly defined
```

Comment:
Separate task, but docs tool should be improved to ignore extra whitespace like this. Doesn't have a noticeable impact on the generated markdown result (at least in GitHub, maybe it does for WordPress handbook).